setup.py

```diff
@@ -1,8 +1,33 @@
-from setuptools import setup, find_namespace_packages
+from setuptools import find_packages, setup
+
+long_description = """ManiSkill2 is a large-scale robotic manipulation benchmark, focusing on learning generalizable robot agents and manipulation skills. It features 2000+ diverse objects, 20 task categories, and a large-scale demonstration set in [SAPIEN](https://sapien.ucsd.edu/), a fully-physical, realistic simulator. The benchmark can be used to study 2D & 3D vision-based imitation learning, reinforcement learning, and motion planning, etc."""
+
+
+def read_requirements():
+    with open("requirements.txt", "r") as f:
+        lines = [l.strip() for l in f.readlines()]
+    install_requires = list(filter(None, lines))
+    return install_requires
+
 
 setup(
     name="mani_skill2",
     version="0.3.0",
-    author="SU Lab at UC San Diego",
-    packages=find_namespace_packages(include=["mani_skill2*", "warp_maniskill*"]),
+    description="ManiSkill2: A Unified Benchmark for Generalizable Manipulation Skills",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    author="ManiSkill2 contributors",
+    url="https://github.com/haosulab/ManiSkill2",
+    packages=find_packages(
+        include=["mani_skill2*", "warp_maniskill*"],
+        exclude=["warp_maniskill.warp.tests"],
+    ),
+    python_requires=">=3.7",
+    install_requires=read_requirements(),
+    # Glob patterns do not automatically match dotfiles
+    package_data={
+        "mani_skill2": ["assets/**", "envs/mpm/shader/**"],
+        "warp_maniskill.warp": ["native/*", "native/nanovdb/*"],
+    },
+    exclude_package_data={"": ["*.convex.stl"]},
```

Comment:
have you tried to package this to the test pypi server yet? is there smth I can test with / build with?

Reply:
If you want to test it locally, you can just `pip install .`